contents/spotlight/startup-puzzle.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "How Puzzle defined its ICP and built a product around it"
+date: 2024-02-01
+author: ["joe-martin"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/startup_puzzle.jpg
+featuredImageType: full
+category: Startups
+---
+
+> Each month, we speak to a team enrolled in PostHog for Startups about the way they work. This time, we spoke to [Puzzle](https://puzzle.io/), an accountancy platform for early-stage startups, about how they define and use their Ideal Customer Persona (ICP). 
+
+Building an Ideal Customer Persona (sometimes known as an Ideal Buyer Profile) is one of the most challenging but essential steps for growing your product. Not only will it help you make product and pricing decisions, it will also clarify your value proposition and dictate how you judge successes and failures.
+
+We’ve previously documented [how we defined our ICP](/founders/creating-ideal-customer-profile), but not all companies approach the task in the same way. So, we spoke to the team at [Puzzle](https://puzzle.io/) about how they defined their ICP, what impact the definition had, and what recommendations they had for others...
+
+## 1. Collect both qualitative and quantitative data
+
+An ICP is, fundamentally, a snapshot of your ideal customer based on a gestalt of many customers. In order to define your ICP you therefore need to start by speaking to your existing users directly and building an intuitive understanding of their needs.
+
+Once you’ve done that, says Scott, you need to pair that intuitive understanding with quantitative data so you can understand if these behaviors exist in isolation or at scale. 
+
+“We’re constantly standing next to founders who are just starting out and I do a healthy amount of interviews,” says Scott. “But it’s one thing for a customer to say they want, need, or plan to do X, Y, or Z — and another thing if they actually do it. That’s where quantitative data is useful. You need both to create your ICP.”
```

Comment:
“We’re constantly standing next to founders at each stage of the startup journey interviewing and learning about their experience,” says Scott. “But it’s one thing for a customer to say they want, need, or plan to do X, Y, or Z — and another thing if they actually do it. That’s where quantitative data is useful. You need both to create your ICP.”

Reply:
```suggestion
“We’re constantly standing next to founders at each stage of the startup journey interviewing and learning about their experience,” says Scott. “But it’s one thing for a customer to say they want, need, or plan to do X, Y, or Z — and another thing if they actually do it. That’s where quantitative data is useful. You need both to create your ICP.”
```